apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -19553,6 +19553,5526 @@
+<e>       <p><l>Elbani<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Elbani<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Eldarov<s n="np"/><s n="cog"/></l>    <r>Eldarov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Eldarova<s n="np"/><s n="cog"/></l>    <r>Eldarova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Elgakayev<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Elgakayev<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Elgakayeva<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Elgakayeva<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Eliasson<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Eliasson<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Elsen<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Elsen<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Elturk<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Elturk<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Emerson<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Emerson<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Emin<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Emin<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Eminefendich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Eminefendich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Eminesku<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Eminesku<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Eminov<s n="np"/><s n="cog"/></l>    <r>Eminov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Eminova<s n="np"/><s n="cog"/></l>    <r>Eminova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Emirajiyev<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Emirajiyev<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Emirajiyeva<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Emirajiyeva<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Emiraliyev<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Emiraliyev<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Emiraliyeva<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Emiraliyeva<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Emirov<s n="np"/><s n="cog"/></l>    <r>Emirov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Emirova<s n="np"/><s n="cog"/></l>    <r>Emirova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Emirsaliyev<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Emirsaliyev<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Emirsaliyeva<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Emirsaliyeva<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Emirvaliyev<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Emirvaliyev<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Emirvaliyeva<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Emirvaliyeva<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
```

Comment:
As discussed, you don't want "mf" with -ov/-ova names, and also don't want both elements of the pair.